install_certificate

```diff
@@ -0,0 +1,87 @@
+#! /bin/bash
+
+proxy_exists(){
+    echo "Check for certificates in $HOME/.globus ... "
+    if [ -d $HOME/.globus ]; then
+        if [ $1 != "N" ] && [ $1 != "y" ]; then
+			echo -e "\tUnknown option \"$1\". You must specify [y/N] to delete or keep any existing .pem files."
+			return 0
+		elif [ -a $HOME/.globus/usercert.pem ] && [ -a $HOME/.globus/userkey.pem ] && [ $1 == "N" ]; then
+            echo -e "\tCertificates exist. Doing nothing."
+            return 0
+        elif [ -a $HOME/.globus/usercert.pem ] && [ -a $HOME/.globus/userkey.pem ] && [ $1 == "y" ]; then
+			echo -e "\tCertificates exist, but removing them in order to replace them."
+			rm -f $HOME/.globus/usercert.pem
+			rm -f $HOME/.globus/userkey.pem
+			return 1
+		fi
+    fi
+    echo -e "\tCouldn't find any certificates."
+    return 1
+}
+
+read -d '' USAGE <<"EOF"
+\\t\\t\\t=================================================================================
+\\t\\t\\tThis script checks if you have globus certificates or lets you
+\\t\\t\\tinstall them from a p12 file (default: mycert.p12)
+\\t\\t\\tTo overwrite existing certificates you must enter specify that [y/N] (default: N)
```

Comment:
seems a bit odd to have y/N in different cases - is there a reason for this?

Reply:
I've seen it in many cases to make sure that the user is making a deliberate choice and will not overwrite the files on a whim. I also didn't want to have to check for multiple cases in bash, so I figured I'd chose a distinct case for each choice. Do you know of a way to make back check 'y' and 'Y' without having to duplicate the conditionals in an || statement?

Reply:
I'm not really sure how choosing distinct cases ensures deliberateness...

You could use some kind of regex thing, but I would just make it lowercase only.

also, "enter specify that" seems redundant somehow